docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -54,10 +54,10 @@ Older versions of ``docker-compose`` do not support all the features required by
 Fetching ``docker-compose.yaml``
 ================================
 
-To deploy Airflow on Docker Compose, you should fetch `docker-compose.yaml <docker-compose.yaml>`__.
-
 .. jinja:: quick_start_ctx
 
+    To deploy Airflow on Docker Compose, you should fetch `docker-compose.yaml <{{ doc_root_url }}docker-compose.yaml>`__.
```

Comment:
This works because it uses the same URL that is on L63 :)

Reply:
<img width="939" alt="image" src="https://user-images.githubusercontent.com/8811558/192621092-3b10e499-4f83-40f2-a9ab-1a4c903ac915.png">